pkg/loader/translate.go

```diff
@@ -57,15 +67,78 @@ func SecondaryBinlogToTxn(binlog *pb.Binlog) (*Txn, error) {
 	return txn, nil
 }
 
-func getColVals(table *pb.Table, cols []*pb.Column) (map[string]interface{}, error) {
+// SecondaryBinlogToOracleTxn translate the Binlog format into Oracle Txn
+func SecondaryBinlogToOracleTxn(binlog *pb.Binlog, tableRouter *router.Table, binlogFilter *bf.BinlogEvent) (*Txn, error) {
```

Comment:
suggest combining this function with the upper one. They are almost the same expect for the router logic.